src/main/java/mate/academy/spring/dto/request/MovieRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
 public class MovieRequestDto {
+    @Size(min = 3, max = 100)
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```

I think it's better to use @NotBlank here too